b2/account_info/in_memory.py

```diff
@@ -89,3 +112,16 @@ def get_minimum_part_size(self):
     @_raise_missing_if_result_is_none
     def get_realm(self):
         return self._realm
+
+    def get_allowed(self):
+        return self._allowed
+
+    @_raise_missing_if_result_is_none
+    def get_allowed_bucket_id(self):
+        allowed = json.loads(self._allowed)
```

Comment:
this will raise a json deserialization error if constructor is called without specifying non-mandatory `allowed`

Reply:
... and remove this function.